2016-11-25-shiny-1.md

```diff
@@ -127,38 +127,38 @@ A basic `app.R` file consists of these parts:
   server <- function(input, output) {}
   ```
 
-- A command to run the app, this should be included at the end of every shiny app. It tells shiny that the user interface comes from the object called `ui` and that the server information (data, plots etc.) comes from the object called `server`.
+- A command to run the app, this should be included at the end of every shiny app. It tells Shiny that the user interface comes from the object called `ui` and that the server information (data, plots etc.) comes from the object called `server`.
 
   ```r
   shinyApp(ui = ui, server = server)
   ```
 
-Create a basic shiny app by copying the bits of code above into your `app.R`, delete any example code generated when you created the file.
+Create a basic shiny app by copying the bits of code above into your `app.R`, delete any example code generated when you created the file. 
```

Comment:
Should this be at the start of this section, where you create the app.R file? Might be just me, but I like reading and copying over into R as I go. So then the order would be create the app.R file, delete the example code, create a basic app by reading about app structure and copying across code.